### PR TITLE
Implements events module

### PR DESCRIPTION
**Pull request by vetyy**
Fixes #ENG-826

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -28,9 +36,9 @@
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
     ],
-    install_requires=["structlog~=21.1.0", "sentry-sdk==1.*", 'contextvars~=2.4;python_version<"3.7"'],
+    install_requires=install_requires,
     extras_require={
-        "tests": ["pytest>=6.2.1"],
+        "tests": ["pytest>=6.2.1", "freezegun==1.1.0"],
         "dev": ["pytest>=6.2.1", "pre-commit>=2.9.3"],
```
Comment (mrknmc) [inline setup.py:42]: dev should be superset of tests in which case you can probably leave `make install` here https://github.com/parrot-com/parrottools/pull/9/files#diff-71cabc4177e41ea8f15a89eb65398fa8187739f68a2762706d84885cd8b2ddc3R20

File: src/parrottools/events/publisher.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+from google.protobuf.json_format import MessageToJson
+from google.protobuf.timestamp_pb2 import Timestamp
+
+logger = logging.getLogger(__name__)
+
+
+class Publisher(abc.ABC):
+    @abc.abstractmethod
+    def publish_event(self, event) -> None:
+        """Called when publishing transcribed block."""
```
Comment (mrknmc) [inline src/parrottools/events/publisher.py:17]:
```suggestion
        """Called when publishing an event."""
```

File: tests/parrottools/events/publisher_test.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import unittest
+from unittest.mock import Mock
+
+from events_monitoring.v1.expectations_pb2 import StartRoughDraft
+from freezegun import freeze_time
+
+from parrottools.events.publisher import KinesisPublisher
+
+
+class TestRecordProcessor(unittest.TestCase):
```
Comment (mrknmc) [inline tests/parrottools/events/publisher_test.py:11]: Can we write this as pytest-style like `tests/parrottools/logging/configure_test.py` ?

**Review by mrknmc — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (vetyy) [reply setup.py:42]: not sure about that: https://github.com/parrot-com/parrottools/runs/3235648811?check_suite_focus=true

File: tests/parrottools/events/publisher_test.py (hunk 3 above)
Comment (vetyy) [reply tests/parrottools/events/publisher_test.py]: done

**Pull request merged**